Source/WebCore/platform/graphics/GraphicsContext.h

```diff
@@ -391,6 +392,10 @@ class GraphicsContext {
     unsigned m_transparencyLayerCount { 0 };
     const IsDeferred m_isDeferred : 1; // NOLINT
     bool m_contentfulPaintDetected : 1 { false };
+
+#if ENABLE(CONTEXT_SAVE_STACK_CAPTURE)
+    Vector<std::unique_ptr<WTF::StackShot>, 1> m_saveStacks;
```

Comment:
StackShot should support automatic allocation, so you might be able to do Vector<WTF::StackShot>